src/protocols/ethereum/components/TransferSendForm.vue

```diff
@@ -31,12 +31,13 @@
             : {},
           enough_coin: [fee.toString(), ETH_COIN_SYMBOL],
         }"
+        @update:model-value="() => shouldUseMaxAmount = false"
```

Comment:
No need to use callback:
```suggestion
        @update:model-value="shouldUseMaxAmount = false"
```